web/beacon-app/src/features/teams/components/NewUserInvitationPage.tsx

```diff
@@ -16,12 +16,14 @@ export function NewUserInvitationPage({ data }: { data: any }) {
 
   const invitee_token = searchParams.get('token');
   const handleSubmitRegistration = (
-    values: NewInvitedUserAccount,
-    helpers: FormikHelpers<NewInvitedUserAccount>
+    values: NewUserAccount,
```

Comment:
Suggestion: We have two different types of new users. One who registers without an invitation and one who registers after receiving an invitation. Would it be clearer to readers of the code to remember the difference when viewing this code by letting this remain as `NewInvitedUserAccount`? 

I see that the file name states `NewUserInvitationPage`, but it may be easy to skip over the file's name.

Reply:
In terms of type definition, if both user types share the same properties, it makes sense to use the same type name to prevent redundant code.

Reply:
Does this mean that the `NewInvitedUserAccount` type should be removed?

Reply:
yes definitely